### ISSUE TITLE
[a11y] 9.1.4.4 Text can be enlarged to 200

### ISSUE THREAD

**Issue by tschiebel** [owncloud/web#10727]
audit: https://infinite.owncloud.com/s/PnaAukHGGsLnrMM

https://bitvtest.de/pruefschritt/bitv-20-web/bitv-20-web-9-1-4-4-text-auf-200-vergroesserbar

Note: For the test, the browser window size of 1280x768 px and zoom magnification of
zoom magnification of 200% was used.

### Personal

- [ ] An enlargement to 200% means that some contents of the page are no longer accessible.
no longer accessible.
![image](https://github.com/owncloud/web/assets/132444655/2f9ff1e0-71ce-4894-954f-8db3d38dc816)

### Shares

- [ ] See commentary on Personal

### Account

- [ ] An enlargement to 200% means that some contents of the page are no longer accessible.
no longer accessible.
![image](https://github.com/owncloud/web/assets/132444655/99d0d6ff-365d-4af2-8416-8eab841db550)

**Comment by pascalwengerter** [comment]
From the `Account` screenshot it seems you're using a rather outdated version of `web`? Just checked the current master and the page looks different and seems fully functional

<img width="1552" alt="Screenshot 2024-04-11 at 10 28 39" src="https://github.com/owncloud/web/assets/16822008/6aa06bb1-3909-46e8-9cba-81ffe75dc3c5">

**Comment by pascalwengerter** [comment]
Ready for a second pair of eyes, not a violation from my POV anymore in the current state of things

**Comment by tbsbdr** [comment]
not a violation from my POV anymore in the current state of things
